examples/docker_provider/Gemfile

```diff
@@ -0,0 +1,5 @@
+source 'http://rubygems.org/'
+
+gem 'test-kitchen'
+gem 'kitchen-terraform'
+gem 'kitchen-dokken'
```

Comment:
That was my intent, forgot to remove kitchen-dokken from the Gemfile. I think we should probably remove that dependency for now and open a new feature for adding kitchen-dokken capability.